blog/releases/announcing-knative-v0-24-release.md

```diff
@@ -0,0 +1,313 @@
+* Optimized generated routes to minimize Envoy configuration size ([net-istio#632](https://github.com/knative-sandbox/net-istio/pull/632))
+* Rename Contonr's ClusterRole and ClusterRoleBinding to differ from existing contour installation ([net-contour#500](https://github.com/knative-sandbox/net-contour/pull/500))
+* Add a new ConfigMap `config-kourier`, with the initial `enable-service-access-logging` setting ([net-kourier#523](https://github.com/knative-sandbox/net-kourier/pull/523))
+
+#### 🐞 Bug Fixes
+
+* Fixed a bug where traffic was briefly routed 'wrong', leading to errors due to exceeded
+queues in deployments with a large activator count and a low service Pod count. ([#11375](https://github.com/knative/serving/pull/11375))
+* Traffic status in Route is updated whenever traffic configuration was wrong. ([#11477](https://github.com/knative/serving/pull/11477))
+* Validates, consistently with other configmaps, that the `_example` section of the features
+ConfigMap is not accidentally modified. ([#11391](https://github.com/knative/serving/pull/11391))
+
+
+### Eventing v0.24
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/v0.24.1 -->
+
+#### 🚨 Breaking or Notable Changes
+
+* You must run the storage migration tool after the upgrade to migrate from v1beta2 to v1 `pingsources.sources.knative.dev` resources. ([#5381](https://github.com/knative/eventing/pull/5381))
+
+#### 💫 New Features & Changes
+
+We're glad to announce that we have introduced a new process to test and develop new features,
+called the [experimental features process](https://github.com/knative/eventing/blob/a574b7ba95b9c8d4743ba3ee12184c39e0415d87/docs/experimental-features.md).
+
+Thanks to this process, you are able to try out the new amazing features and provide feedback back to the project!
+
+We're introducing two experimental features to begin with:
+
+* `kreference-group`: When using `ref`, refer to resources using only the API Group, and not the full API Version.
+* `delivery-timeout`: When using the `delivery`, define per request timeout.
+
+You can read more about how to enable these features and their usage in the
+[experimental feature documentation](https://dev-knative.netlify.app/development/eventing/experimental-features/).
+
+* `KReference.Group` now can be used in `Subscription.Spec.Channel` as well ([#5520](https://github.com/knative/eventing/pull/5520))
+* Added `DeliverySpec.Timeout` ([#5149](https://github.com/knative/eventing/pull/5149))
+* Added the experimental feature kreference-group.
+By enabling it, you can use Subscriber.Ref.Group instead of Subscriber.Ref.APIVersion to refer to another Resource, without being explicit about the resource version (for example, v1beta1, v1, ...) ([#5440](https://github.com/knative/eventing/pull/5440))
```

Comment:
```suggestion
By enabling it, you can use `Subscriber.Ref.Group` instead of `Subscriber.Ref.APIVersion` to refer to another Resource, without being explicit about the resource version (for example, v1beta1, v1, ...) ([#5440](https://github.com/knative/eventing/pull/5440))
```